packages/x-data-grid/src/hooks/utils/useGridAriaAttributes.tsx

```diff
@@ -16,8 +16,17 @@ export const useGridAriaAttributes = (): React.HTMLAttributes<HTMLElement> => {
   const headerGroupingMaxDepth = useGridSelector(apiRef, gridColumnGroupsHeaderMaxDepthSelector);
   const pinnedRowsCount = useGridSelector(apiRef, gridPinnedRowsCountSelector);
 
+  const generatedLabelId = 'data-grid-label';
```

Comment:
We need to generate a label ID that can be accessed from this hook, and the `<ToolbarLabel />` component

Reply:
Opted to not use `aria-labelledby` and just use `aria-label` in all circumstances